src/scripts/main.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
-// write code here
+const WALL = document.querySelector('.wall');
+const SPIDER = document.querySelector('.spider');
+
+WALL.style.position = 'relative';
+SPIDER.style.top = '170px';
+SPIDER.style.left = '170px';
```

Comment:
it's bad practice to hardcore pixels, it's also bad for responsive
<img width="337" alt="image" src="https://github.com/mate-academy/js_center_spider_DOM/assets/70220667/9903db4e-9f11-4058-b831-a13b99818b6c">